keyboards/boston_meetup/2019/2019.c

```diff
@@ -0,0 +1,226 @@
+/* Copyright 2019 ishtob
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "2019.h"
+#include "qwiic.h"
```

Comment:
Why are these necessary? 

In fact, in theory, only the 2019.h should be necessary. Everything else *should* be handled by quantum.h.

Reply:
not sure why, but there are needed to not get error during compiling, looks like these codes calls some function from those

Reply:
Ah, then it's likely because they weren't properly included in quantum.h, then. :(

Then it's fine, for now.